pkg/services/screenshot/screenshot.go

```diff
@@ -216,11 +152,11 @@ func NewObservableScreenshotService(r prometheus.Registerer, service ScreenshotS
 			Namespace: namespace,
 			Subsystem: subsystem,
 		}),
-		failures: promauto.With(r).NewCounter(prometheus.CounterOpts{
+		failures: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
 			Name:      "failures_total",
 			Namespace: namespace,
 			Subsystem: subsystem,
-		}),
+		}, []string{"reason"}),
```

Comment:
should we add Dashboard and panel IDs to the labels? I think it would help troubleshooting in the case of panel rendering problems

Reply:
Do we need to be concerned about the total number of active series with those two extra dimensions?